h/search/index.py

```diff
@@ -63,7 +58,6 @@ def index(
         indexing = es_helpers.streaming_bulk(
             self.es_client.conn,
             annotations,
-            chunk_size=chunk_size,
```

Comment:
`chunk_size=ES_CHUNK_SIZE`

And ditto for `PG_WINDOW_SIZE`, I can't see the caller ever wanting / being trusted to change this value.